From ca752ccc7831a2d62fc86971c28eea2fd037d0d2 Mon Sep 17 00:00:00 2001
From: Manuel Transfeld <auipga@users.noreply.github.com>
Date: Fri, 25 Jun 2021 17:47:37 +0200
Subject: [PATCH 151/153] Default shortcut Super+t

Change the default shortcut from <kbd>Super</kbd>+<kbd>t</kbd> to <kbd>Super</kbd>+<kbd>h</kbd> to avoid conflicts with the shortcut for Gnome minimize window

(cherry picked from commit 15ffb927fa601a194d44d06741c5f3e1a0633c97)
---
 .../org.gnome.shell.extensions.project-hamster.gschema.xml      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/schemas/org.gnome.shell.extensions.project-hamster.gschema.xml b/data/schemas/org.gnome.shell.extensions.project-hamster.gschema.xml
index 21a08d1..8a35b28 100644
--- a/data/schemas/org.gnome.shell.extensions.project-hamster.gschema.xml
+++ b/data/schemas/org.gnome.shell.extensions.project-hamster.gschema.xml
@@ -14,7 +14,7 @@
         </key>
 
         <key type="as" name="show-hamster-dropdown">
-            <default><![CDATA[['<Super>h']]]></default>
+            <default><![CDATA[['<Super>t']]]></default>
             <summary>Global key combination to show the drop-down.</summary>
         </key>
     </schema>
-- 
2.35.1

